docs/tutorials/designate.md

```diff
@@ -28,16 +28,19 @@ way to get yourself an OpenStack installation to play with is to use [DevStack](
 
 ## Creating DNS zones
 
-All domain names that are ExternalDNS is going to create must belong to one of DNS zones created in advance. Here is an example of how to create `example.com` DNS zone:
+All domain names that are ExternalDNS is going to create must belong to one of DNS zones created in advance. Here is an example of how to create `external-dns-test.my-org.com` DNS zone:
```

Comment:
There is still a typo in this sentence `All domain names that are ExternalDNS is`. 

Also, why are we changing the domain for the examples?

Reply:
I fixed the typo in the sentence above. The main idea was include an example for RBAC in the designate tutorial. As most Kubernetes clusters use RBAC so the examples will not work. I decided to use the same domain used in the aws tutorial.